scripts/cluster_launcher.py

```diff
@@ -98,7 +98,7 @@ def parseJobsFile(self, template_dir, job_file):
         return jobs
 
     def createAndLaunchJob(self, template_dir, job_file, specs, options):
-        next_dir = getNextDirName(specs['job_name'], os.listdir('.'))
+        next_dir = getNextDirName(specs['job_name'], os.listdir(template_dir))
```

Comment:
Good catch, I'm surprised this didn't bite us earlier. I guess I always assumed we'd be in the right directory.